web3j/src/main/resources/config/application.yml

```diff
@@ -1,6 +1,6 @@
 workload:
   model: OPEN  # either OPEN or CLOSED workload model
-  transactions: 1_000 # total transactions to be done
+  transactions: 1_00 # total transactions to be done
```

Comment:
The _ group separator with 100 looks semantically incorrect. I would suggest revert back to 1_000.